Soomla/Assets/Plugins/Soomla/Profile/SoomlaProfile.cs

```diff
@@ -882,7 +890,12 @@ public class SoomlaProfile
 		}
 
 		internal static void TryFireProfileInitialized () {
-			if (AllProvidersInitialized()) {
+			if (AllProvidersInitialized()
+#if !UNITY_EDITOR
+				&& nativeModulesInitialized
```

Comment:
@EugeneButusov I just thought of a few options:
1. `internal static bool nativeModulesInitialized = true;`
2. in Initialize after `instance._initialize(GetCustomParamsJson(customParams));`
3. Before calling `TryFireProfileInitialized()`
Wherever you pick. If you think of another place, go for it.